.circleci/config.yml

```
@@ -8,8 +8,10 @@ workflows:
       - lint
       - acceptance-test:
           requires: [test, lint]
-      - push:
+      - cypress:
           requires: [acceptance-test]
+      - push:
+          requires: [cypress]
```

CircleCI would be a little bit faster running acceptance-test and cypress in parallel:
```suggestion
      - cypress:
          requires: [test, lint]
      - push:
          requires: [acceptance-test, cypress]
```